Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
It's partially fixed then. In old revisions, without Safe Texture Cache, the intro logo, the loading screens and all dancers were erratic (dropping frames), setting Safe Texture Cache to normal or safe fixed that. In current development builds, only the first frame of the logo is dropped, everything else remains fine (including the loading screens and the dancers). I would leave Safe Texture Cache option out as it decreases performance a little just to fix a very small artifact in the intro logo...